src/emm/emm_cli.py

```diff
@@ -211,5 +220,15 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_context_width=100))
+@click.argument("commit", nargs=-1)
+@click.option("--directory", default=None, help="Directory to execute command at.")
+@click.pass_context
+def git_commit(ctx: click.Context, commit: str, directory: str) -> None:
+    """Perform git commit -a operation to all modules."""
```

Comment:
Can we expand this description? I think we should provide a summary (something like "Perform a git command of all modules"), followed by a more detailed description of what is happening, including that the `--all` flag will be used when creating the commits.